AppiumForMac/Server/Controller/AfMSessionController.m

```diff
@@ -891,6 +891,55 @@ -(PFUIElement*) windowForHandle:(NSString*)windowHandle
 	return [windows objectAtIndex:windowIndex];
 }
 
+-(BOOL) scrollPage:(id)element
+{
+    float increment_by = 0.25;
+    bool scroll_found = false;
+    PFUIElement* scroll;
+    float axValue = 0,adder = 0;
+
+    id parent = [(PFUIElement*)element AXParent];
+    NSArray* siblings = [parent AXChildren];
+
+    while(scroll == nil && parent != nil)
```

Comment:
please add space after while, if and other similar keywords to match with the common project style